### PR TITLE
ToolbarV3 - more button

### PR DESCRIPTION
**Pull request by justame**

### CODE REVIEW

File: packages/toolbars-v3/web/src/components/SizeCalculator/SizeCalculator.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      });
+
+      elements.forEach((element, index) => {
+        if (sumVisible + element.width <= width) {
+          visible.push(components[index]);
+          sumVisible += element.width;
+        } else {
+          overflowed.push(components[index]);
+        }
+      });
```
Comment (noamtwix) [inline packages/toolbars-v3/web/src/components/SizeCalculator/SizeCalculator.tsx:45]:
```suggestion
      elements.forEach((element, index) => {
        if (sumVisible + element.width <= width) {
          visible.push(components[index]);
        } else {
          overflowed.push(components[index]);
        }
        sumVisible += element.width;
      });
```
there can be a strange behavior without this change (I know from my PR)

File: packages/toolbars-v3/web/src/components/Toolbar/Toolbar.tsx [hunk 2]
```diff
@@ -57,7 +58,11 @@ class ToolbarComponent extends Component<ToolbarProps, Record<string, unknown>>
             return <div />;
           }
           return (
-            <div data-hook="toolbar-v3" className={styles.toolbar}>
+            <div
+              dir="ltr"
```
Comment (justame) [inline packages/toolbars-v3/web/src/components/Toolbar/Toolbar.tsx:62]: why do we need dir?

**Comment by justame** [comment]
Looks good, feel free to merge

File: packages/toolbars-v3/web/src/components/Toolbar/Toolbar.tsx (hunk 2 above)
Comment (noamtwix) [reply packages/toolbars-v3/web/src/components/Toolbar/Toolbar.tsx:62]: to make the 'more button modal' `right: 0` CSS prop work

**Comment by noamtwix** [comment]
merge without the `sumVisible += element.width;` change?

**Comment by justame** [comment]
Yes, merge without the change

**Pull request merged**